packages/pigeon/platform_tests/test_plugin/android/build.gradle

```diff
@@ -65,7 +65,7 @@ android {
     }
 
     dependencies {
-        testImplementation 'junit:junit:4.+'
+        testImplementation 'junit:junit:4.13.2'
```

Comment:
This causes the [GradleDynamicVersion](https://googlesamples.github.io/android-custom-lint-rules/checks/GradleDynamicVersion.md.html)